src/db/connection.js

```diff
@@ -0,0 +1,43 @@
+const mongoose = require('mongoose');
+
+const url = process.env.MONGODB_ATLAS_URL;
+
+class Connection {
```

Comment:
@emirsagit you can do the connection the normal way there's no need to do it in a singleton pattern we did it in class just as a demonstration of singleton

Reply:
@Ammar-64 isn't singleton pattern better in general? or just in special cases where you have to connect multiple times?

Reply:
In general, singleton might be better if we're using mongoDB for example, but in this case, we're using mongoose built-in method which already uses some kind of pattern under the hood

Reply:
Okay thanks Ammar, @emirsagit sorry for the extra work (but we learnt something), you can just revert the commit and do the other edits.

Reply:
No problem @alfawal we've learned sth. I've changed the connection to the normal way.